src/Symfony/Component/HttpClient/RetryableHttpClient.php

```diff
@@ -77,14 +77,17 @@ public function request(string $method, string $url, array $options = []): Respo
 
                     if (false === $shouldRetry) {
                         $context->passthru();
+
                         if (null !== $firstChunk) {
                             yield $firstChunk;
-                            yield $context->createChunk($content);
-                            yield $chunk;
-                        } else {
-                            yield $chunk;
                         }
-                        $content = '';
+
+                        while ('' !== $c = $content) {
```

Comment:
isn't this just a `if ($content !== '')` ?

Reply:
I reverted this change which was part of a previous attempt at fixing the issue.